<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Notification</class>
 <widget class="QDialog" name="GUI_Notification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>185</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Notification settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Timeout may depend on server settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cb_activate">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Activate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btn_ok">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="combo_notification">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sb_timeout">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>5000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Timeout (ms)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Notifications</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
